tasks_management/services.py

```diff
@@ -0,0 +1,49 @@
+import logging
+
+from django.db import transaction
+
+from core.services import create_or_update_core_user, BaseService
+from core.services.utils import check_authentication, output_exception, model_representation, output_result_success
+from tasks_management.models import TaskGroup, TaskExecutor
+from tasks_management.validation import TaskGroupValidation, TaskExecutorValidation
+
+logger = logging.getLogger(__name__)
+
+
+class TaskGroupService:
+    OBJECT_TYPE = TaskGroup
+
+    def __init__(self, user):
+        self.user = user
+        self.validation_class = TaskGroupValidation
+
+    @check_authentication
+    def create(self, obj_data):
+        try:
+            with transaction.atomic():
```

Comment:
Transaction can be added as wrapper on the create method.

Reply:
Should we also change that in BaseService?
![image](https://github.com/openimis/openimis-be-tasks_management_py/assets/58431955/d0ead146-f380-4c3f-91bb-3aebc8b6f5a1)

Reply:
There's no need for that, it's about the syntax and functionality should remain the same.